pkg/controllers/qat/controller_test.go

```diff
@@ -123,18 +123,48 @@ func (c *controller) newDaemonSetExpected(rawObj client.Object) *apps.DaemonSet
 			},
 		},
 	}
+
+	// add the optional init container
+	if devicePlugin.Spec.InitImage != "" {
+		setInitContainer(&daemonSet.Spec.Template.Spec, devicePlugin.Spec)
+	}
```

Comment:
I think the `controller_test.go` changes are not needed here.

Reply:
Ok. changed back!